packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx

```diff
@@ -86,6 +89,12 @@ const WorkspaceHeader: React.FC<{}> = () => {
     history.replace("/login");
   }, [history, apolloClient]);
 
+  const handleUpgradeClick = useCallback(() => {
+    trackEvent({
+      eventName: AnalyticsEventNames.UpgradeOnTopBarClick,
```

Comment:
According to the Notion docs you need to add this metadata:
Reason: “Top panel CTA”
Location: App
Workspace: workspace id
User: user id